src/tools/gophish_import.py

```diff
@@ -281,12 +282,12 @@ def main() -> None:
         with open(args["ASSESSMENT_FILE"]) as json_file:
             assessment = json.load(json_file)
     except FileNotFoundError as e:
-        logging.error(f"{e}\n")
+        logging.exception("%s", e)
         # Stop logging and clean up
         logging.shutdown()
         sys.exit(1)
     except PermissionError as e:
-        logging.error(f"{e}\n")
+        logging.exception("%s", e)
```

Comment:
Now that you are using `logging.exception(), all the exception information will appear in the stack trace; hence it would be better to do something like `logging.exception("Unable to write to assessment file")` (or whatever the message needs to be to be descriptive and useful) instead of simply `logging.exception("%s", e)`.  Just a suggestion that could be applied to most all of the `logging.exception()` calls.

Reply:
I added some more verbose messaging to the exception. It should hopefully give a bit more guidance about what was happening prior to the exception. 

See 3ee423033fefa9873fd13190c0e6edb575cd0080